test/unit/events.test.js

```diff
@@ -1,121 +1,133 @@
+/* eslint-env qunit */
 import * as Events from '../../src/js/utils/events.js';
 import document from 'global/document';
 
-q.module('Events');
+QUnit.module('Events');
 
-test('should add and remove an event listener to an element', function(){
-  expect(1);
+QUnit.test('should add and remove an event listener to an element', function() {
+  QUnit.expect(1);
 
-  var el = document.createElement('div');
-  var listener = function(){
-    ok(true, 'Click Triggered');
+  const el = document.createElement('div');
+  const listener = function() {
+    QUnit.ok(true, 'Click Triggered');
   };
 
   Events.on(el, 'click', listener);
-  Events.trigger(el, 'click'); // 1 click
+  Events.trigger(el, 'click');
+  // 1 click
```

Comment:
these comments (this line and line 20) should go above the line they were associated with.